### ISSUE TITLE
No way to inspect errors in IPythonViz

### ISSUE THREAD

**Issue by tbekolay** [nengo/nengo_gui#557]
My sim starts in Nengo, but not when I make a NengoGUI instance with IPythonViz; the play button turns to an X.

![nengogui](https://cloud.githubusercontent.com/assets/203709/9969504/de6ce96e-5e1c-11e5-84f7-79a228df1d0e.png)

If I weren't in the notebook, I believe that I could inspect the error in the console below the source code editor. However, since I'm in the notebook, there's no way to see what the error is.

Not sure what the best way to display this would be. Maybe the console should still be visible when in the notebook? Or a tooltip on hovering over the X?

**Comment by Seanny123** [comment]
Can't we just display the error in the same way most iPython errors are shown?

**Comment by jgosmann** [comment]
That would be as an exception in the output cell? No because that requires a synchronous execution pattern.

**Comment by tcstewar** [comment]
Hmm...  would it make sense to just modify the AceEditor such that it'll automatically pop itself open if the error message gets set?  That seems like a useful feature in general and should solve this problem too....

**Comment by jgosmann** [comment]
But there is no code to display ...

**Comment by jgosmann** [comment]
or is there?

**Comment by tcstewar** [comment]
> But there is no code to display ...

So the code window will be blank, but the area below that that shows the error will still be shown.

**Comment by jgosmann** [comment]
Yeah, but wouldn't a blank code window be strange? Though, it might be better for now than not displaying errors at all.

**Comment by tcstewar** [comment]
> Yeah, but wouldn't a blank code window be strange? Though, it might be better for now than not displaying errors at all.

Better than not displaying the error at all, I think.  Although it might not be too much extra effort to have it automatically resize the code window to be zero height.....

**Comment by tcstewar** [comment]
I'm now thinking a better option would be to send that error to std.error and allow Jupyter to show it instead.  That might be a quick thing to try implementing....
